routes/game.rb

```diff
@@ -145,7 +145,19 @@ class Api
               halt(400, 'Player count not supported')
             end
 
-            set_game_state(game, engine, users)
+            acting = set_game_state(game, engine, users)
+
+            unless acting.empty?
+              MessageBus.publish(
```

Comment:
game_url looks changed here maybe you should turn this into a function

Reply:
Indeed. I don't know how that could have happened (there was no change on this line..)